docs/architecture/README.md

```diff
@@ -29,71 +29,73 @@ Note the context/background should be written in the present tense.
+- [ADR-005: Consensus-Params](./adr-005-consensus-params.md)
+- [ADR-008: Priv-Validator](./adr-008-priv-validator.md)
 - [ADR-009: ABCI-Design](./adr-009-ABCI-design.md)
+- [ADR-010: Crypto-Changes](./adr-010-crypto-changes.md)
+- [ADR-011: Monitoring](./adr-011-monitoring.md)
 - [ADR-014: Secp-Malleability](./adr-014-secp-malleability.md)
 - [ADR-015: Crypto-Encoding](./adr-015-crypto-encoding.md)
+- [ADR-016: Protocol-Versions](./adr-016-protocol-versions.md)
+- [ADR-017: Chain-Versions](./adr-017-chain-versions.md)
+- [ADR-018: ABCI-Validators](./adr-018-ABCI-Validators.md)
+- [ADR-019: Multisigs](./adr-019-multisigs.md)
 - [ADR-020: Block-Size](./adr-020-block-size.md)
+- [ADR-021: ABCI-Events](./adr-021-abci-events.md)
 - [ADR-025: Commit](./adr-025-commit.md)
+- [ADR-026: General-Merkle-Proof](./adr-026-general-merkle-proof.md)
+- [ADR-033: Pubsub](./adr-033-pubsub.md)
 - [ADR-034: Priv-Validator-File-Structure](./adr-034-priv-validator-file-structure.md)
-- [ADR-039: Peer-Behaviour](./adr-039-peer-behaviour.md)
+- [ADR-043: Blockchain-RiRi-Org](./adr-043-blockchain-riri-org.md)
 - [ADR-044: Lite-Client-With-Weak-Subjectivity](./adr-044-lite-client-with-weak-subjectivity.md)
 - [ADR-046: Light-Client-Implementation](./adr-046-light-client-implementation.md)
 - [ADR-047: Handling-Evidence-From-Light-Client](./adr-047-handling-evidence-from-light-client.md)
+- [ADR-051: Double-Signing-Risk-Reduction](./adr-051-double-signing-risk-reduction.md)
+- [ADR-052: Tendermint-Mode](./adr-052-tendermint-mode.md)
 - [ADR-053: State-Sync-Prototype](./adr-053-state-sync-prototype.md)
+- [ADR-054: Crypto-Encoding-2](./adr-054-crypto-encoding-2.md)
 - [ADR-055: Protobuf-Design](./adr-055-protobuf-design.md)
 - [ADR-056: Light-Client-Amnesia-Attacks](./adr-056-light-client-amnesia-attacks)
 - [ADR-059: Evidence-Composition-and-Lifecycle](./adr-059-evidence-composition-and-lifecycle.md)
+- [ADR-062: P2P-Architecture](./adr-062-p2p-architecture.md)
+- [ADR-063: Privval-gRPC](./adr-063-privval-grpc.md)
+- [ADR-066-E2E-Testing](./adr-066-e2e-testing.md)
+### Accepted
+
+- [ADR-006: Trust-Metric](./adr-006-trust-metric.md)
+- [ADR-024: Sign-Bytes](./adr-024-sign-bytes.md)
+- [ADR-035: Documentation](./adr-035-documentation.md)
+- [ADR-039: Peer-Behaviour](./adr-039-peer-behaviour.md)
 - [ADR-060: Go-API-Stability](./adr-060-go-api-stability.md)
 - [ADR-061: P2P-Refactor-Scope](./adr-061-p2p-refactor-scope.md)
-- [ADR-062: P2P-Architecture](./adr-062-p2p-architecture.md)
 - [ADR-065: Custom Event Indexing](./adr-065-custom-event-indexing.md)
-- [ADR-066-E2E-Testing](./adr-066-e2e-testing.md)
 - [ADR-068: Reverse-Sync](./adr-068-reverse-sync.md)
 - [ADR-067: Mempool Refactor](./adr-067-mempool-refactor.md)
 
 ### Rejected
 
+- [ADR-023: ABCI-Propose-tx](./adr-023-ABCI-propose-tx.md)
+- [ADR-028: libp2p](./adr-028-libp2p.md)
 - [ADR-029: Check-Tx-Consensus](./adr-029-check-tx-consensus.md)
+- [ADR-031: Changelog-structure](./adr-031-changelog.md)
```

Comment:
Lol I find these empty ADR's to be a bit pointless. I guess it illustrates that there was a conversation / decision but having no reason why the outcome was decided or all the points of interest is a bit frustrating.

Reply:
Same, I would prefer to delete, but then the question arises why was adr-031 deleted.

Reply:
How was it made in the first place? Is there no evidence of any more information?

Reply:
oh, it was a placeholder. Before numbers were assigned prior to work through an issue, the adr never got written. When I closed the issue I opened blank adrs instead of changing numbers and potentially breaking links.

Reply:
I can't imagine there being too many things out there linking to an ADR that was never written 😄 - Personally I would just delete them. People would probably be more confused at opening an empty ADR then getting a 404 but I also don't see too much trouble in just leaving them there

Reply:
You convinced me 😉 Ill remove them